### ISSUE TITLE
Loss for multi-variate LSTM Model is decreasing very slowly.

### ISSUE THREAD

**Issue by Savvy98** [keras-team/keras#13729]
Hello,
I'll keep it short. I'm building a Multi-Variate LSTM Model (12 Features, wanting to detect the first). The loss decreases very slowly and the rmse gets worse with more the data I train with.

`model = Sequential()`
`model.add(LSTM(50, input_shape=(train_X.shape[1], train_X.shape[2])))`
`model.add(Dense(1))`
`model.compile(loss='mean_squared_error', optimizer='adam')`
`history = model.fit(train_X, train_y, epochs=200, batch_size=10, validation_data=(valid_train_X, valid_train_y), verbose=1, shuffle=False)`

#In the beginning 
Train on 58281 samples, validate on 6476 samples
Epoch 1/200
58281/58281 [==============================] - 16s 278us/step - loss: 0.0023 - val_loss: 0.0030
Epoch 2/200
58281/58281 [==============================] - 15s 263us/step - loss: 0.0021 - val_loss: 0.0029

#And after "200 epochs"
58281/58281 [==============================] - 15s 263us/step - loss: 0.0019 - val_loss: 0.0029
Epoch 200/200
58281/58281 [==============================] - 15s 262us/step - loss: 0.0019 - val_loss: 0.0029

I really need some help in optimizing this model. If you give me any tips, I'll test it out right away and show you the results asap
Thanks in advance!

**Comment by rsreeram17** [comment]
Were you able to solve this issue? I am also facing the same issue for my problem

**Comment by sushreebarsa** [comment]
@Savvy98 
In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!
